AVIenabledbyArc/vi-edge-deployment-script.sh

```diff
@@ -235,18 +295,13 @@ if [[ $install_extension == "true" ]]; then
                           --auto-upgrade-minor-version false \
                           --config-protected-settings "speech.endpointUri=${speechEndpoint}" \
                           --config-protected-settings "speech.secret=${speechPrimaryKey}" \
-                          --config-protected-settings "translate.endpointUri=https://eastus.api.cognitive.microsoft.com" \
+                          --config-protected-settings "translate.endpointUri=${translatorEndpoint}" \
                           --config-protected-settings "translate.secret=${translatorPrimaryKey}" \
+                          --config-protected-settings "ocr.endpointUri=${ocrEndpoint}" \
+                          --config-protected-settings "ocr.secret=${ocrPrimaryKey}"\
                           --config "videoIndexer.accountId=${viAccountId}" \
-                          --config "frontend.endpointUri=https://${EXTERNAL_IP}" \
+                          --config "frontend.endpointUri=https://${ENDPOINT_URI}" \
                           --config AI.nodeSelector."beta\\.kubernetes\\.io/os"=linux \
```

Comment:
for the AI it is important for them to see what values are needed to be changed if they will want to use diffrent selectors.
otherwise finding this lable AI.nodeSelector is quite hard.